src/main/java/org/homeschoolpebt/app/cli/SftpClientImpl.java

```diff
@@ -0,0 +1,43 @@
+package org.homeschoolpebt.app.cli;
+
+import com.jcraft.jsch.Channel;
+import com.jcraft.jsch.ChannelSftp;
+import com.jcraft.jsch.JSch;
+import com.jcraft.jsch.JSchException;
+import com.jcraft.jsch.Session;
+import com.jcraft.jsch.SftpException;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+@Component
+@Profile("!test")
+public class SftpClientImpl implements SftpClient {
+
+  String username;
+  String password;
+  String uploadUrl;
+
+  public SftpClientImpl(@Value("${sftp.username:}") String username, @Value("${sftp.password:}") String password, @Value("${sftp.upload-url:}") String uploadUrl) {
+    this.username = username;
+    this.password = password;
+    this.uploadUrl = uploadUrl;
+  }
+
+  @Override
+  public void uploadFile(String zipFilename) throws JSchException, SftpException {
+    JSch jsch = new JSch();
+    jsch.setKnownHosts("~/.ssh/known_hosts");
```

Comment:
Dose this still work even if this file doesn't exist? Maybe we should actually make that file as part of our build process. The value I have for it is:

```
mft.safe.cdt.ca.gov ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCi/KKNiEmAyBT6lmCJ+pzIusbEdYbN7wc3mt6dQz9SDbHRtpNWbRFdsKSyzjNZr3Br92Jl5IxqW1Hjdfa3w66NuyFs/jsyRqrsYZIpn/nxDsPJTXxPCJ2c7Qy34A0O6c2I1w7Imss78Ddc63tlFLaMnL3WuHBRHzt2oneT0rqSn0QFrTUWzz3a3ojMTRksARopyraQ5nJKe1nm1QqbN9NoLZG90aveSACLPJ8RvfMMUTKIW73tFk3UHuOO+6Z7LrEKVuIGLWGuwLK/fyJInjcpKtRV2mwE4dv5UOLGmcBIGk6kU0voZkPUegrG86tVHT7+42AJ0J5gBb4jCVRT7yPV
```